quickstart.rst

```diff
@@ -153,7 +153,10 @@ Starting a Flux Instance
 
 In order to use Flux, you first must initiate a Flux *instance* or *session*.
 
-A Flux session is composed of a hierarchy of ``flux-broker`` processes which are launched via any parallel launch utility that supports PMI. For example, ``srun``, ``mpiexec.hydra``, etc., or locally for testing via the ``flux start`` command.
+A Flux session is composed of a hierarchy of ``flux-broker`` processes
+which are launched via any parallel launch utility that supports PMI. For
+example, ``srun``, ``mpiexec.hydra``, etc., or locally for testing via the
+``flux start`` command with the ``-s, --test-size=N`` option.
```

Comment:
Sounds good! I should have mentioned I wrapped all lines in quickstart to ~78 as well, so it may look like more was changed that actually was in this PR.

I dislike 1 line per paragraph in text documents under version control since `diff` is mostly line based, so I thought I'd fix that now.